backend/Agent/quiz/agent.py

```
@@ -0,0 +1,863 @@
+class QuizAgent(BaseAgent):
+    """퀴즈 생성 및 관리 Agent"""
+
+    def __init__(self):
+        super().__init__(agent_type="quiz")
+        self.openai_client = OpenAIClient(model="gpt-4o-mini")
+        self.vector_client = VectorClient()
+        self.mongodb_client = MongoDBClient()
+
+    async def process(
+        self,
+        user_input: str,
```

Ruff (0.14.5)

Unused method argument: `user_input`

(ARG002)